commit 26e3e557e25b0ba5103dfb683050d2435f7708bb
Author: Ankit Jain <ankit.jain@xamarin.com>
Date:   Wed Feb 8 18:44:41 2017 -0500

    [msbuild] Add support for .mdb files to be copied, with mono's msbuild

    msbuild defaults to .pdb files as the default debug file extension.
    Mono's msbuild has `$(_DebugFileExt)` property to override this. We need
    to set that to `.mdb`, else the generated `.mdb` files won't get
    copied/deployed.

    But this feature is not upstream yet, so it is not present in upstream
    F# yet.

diff --git a/src/fsharp/FSharp.Build/Microsoft.FSharp.targets b/src/fsharp/FSharp.Build/Microsoft.FSharp.targets
index 82125d2..e7ddac2 100644
--- a/src/fsharp/FSharp.Build/Microsoft.FSharp.targets
+++ b/src/fsharp/FSharp.Build/Microsoft.FSharp.targets
@@ -40,6 +40,10 @@ Copyright (C) Microsoft Corporation. Apache 2.0 License.
         <RootNamespace Condition="'$(RootNamespace)'==''">RootNamespace</RootNamespace>
         <Actual32Bit Condition="'$(TargetFrameworkVersion)'=='v2.0' or '$(TargetFrameworkVersion)'=='v3.0' or '$(TargetFrameworkVersion)'=='v3.5' or '$(TargetFrameworkVersion)'=='v4.0'">false</Actual32Bit>
         <Actual32Bit Condition="!('$(TargetFrameworkVersion)'=='v2.0' or '$(TargetFrameworkVersion)'=='v3.0' or '$(TargetFrameworkVersion)'=='v3.5' or '$(TargetFrameworkVersion)'=='v4.0')">$(Prefer32Bit)</Actual32Bit>
+
+        <!-- _DebugFileExt is not an upstream msbuild feature yet -->
+        <_DebugFileExt Condition="'$(FscDebugFileExt)' != ''">$(FscDebugFileExt)</_DebugFileExt>
+        <_DebugFileExt Condition="'$(_DebugFileExt)' == ''">.mdb</_DebugFileExt>
     </PropertyGroup>

     <!--
